cultivation/scripts/running/fatigue_watch.py

```
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+"""
+Scan subjective.csv → raise GitHub Issue+label when any red-flag rule breaches.
+
+Run nightly via cron Action OR manual GH-CLI.
+"""
+import os, json, datetime as dt
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import

(F401)